src/Resources/config/monsieurbiz/settings.yaml

```
@@ -0,0 +1,12 @@
+monsieurbiz_sylius_settings:
+    plugins:
+        monsieurbiz.nocommerce:
+            vendor_name: Monsieur Biz
+            vendor_url:
+            plugin_name: No Commerce
+            description: No Commerce configuration
+            icon: shopping bag
+            use_locales: false
```

```suggestion
            use_locales: true
```

We could accept to have a shop only for a specific locale, right?

---

yes, it's possible to want disable firewalls only for a locale